civihr_employee_portal/templates/civihr-employee-portal-custom-reports.tpl.php

```diff
@@ -105,6 +105,9 @@ class="btn btn-lg btn-secondary-outline text-uppercase">
             </div>
         </footer>
     </section>
+    <div data-graph-section="summary">
```

Comment:
Nah, if it's to be done, for now it's via vanilla javascript with the hope that in the future we can plugin angular